products/admin.py

```diff
@@ -23,3 +22,23 @@ class Brand(admin.ModelAdmin):
 @admin.register(BrandProduct)
 class BrandProduct(admin.ModelAdmin):
     list_display = ('brand', 'product')
+
+
+@admin.register(Section)
+class Section(admin.ModelAdmin):
+    list_display = ('name', 'order', 'is_active', 'section_file')
+
```

Comment:
@rinkishivani give proper name to admin classes. e.g SectionAdmin, SectionItemsAdmin, etc. and add inline admin class for through model.
reference: https://medium.com/nerd-for-tech/from-zero-to-hero-django-admin-model-relations-part5-c4c60c6d57a7
cc: @lalitvasoya

Reply:
Done